packages/@react-stately/table/src/useTableColumnResizeState.ts

```diff
@@ -47,11 +50,14 @@ export interface TableColumnResizeState {
   /** Gets the current maxWidth for the specified column. */
   getColumnMaxWidth: (key: Key) => number,
   /** Currently calculated widths for all columns. */
-  widths: Map<Key, number>
+  widths: Map<Key, number>,
+  /** Key of the currently resizing column. */
+  resizingColumn: Key | null,
+  tableState: TableState<T>
```

Comment:
Why is this in two places, both in the props and as a separate argument?

Reply:
I don't follow, where do you see it in props? this is the return object of the hook

Reply:
oh sorry. I mis-read.